services/crypto_service.proto

```diff
@@ -58,7 +58,7 @@ service CryptoService {
     rpc approveAllowances (Transaction) returns (TransactionResponse);
 
     /**
-     * Deletes the approved hbar or token allowances on an owner account.
+     * Deletes the approved NFT allowances on an owner account.
```

Comment:
Might be better to mention `one or more specific NFT serial numbers` to avoid confusion of `approvedForAll`

Reply:
thanks updated